docs/deployments/deploy-to-vercel.mdx

```diff
@@ -19,21 +19,21 @@ Let's walk through the following:
 
 To set up a new Vercel project, start by [going to your Vercel Dashboard](https://vercel.com/dashboard). Here, you'll see a list of your projects or, if you don't have any projects yet, a prompt to create a new project:
 
-![A Vercel dashboard with a "No projects, yet!" message. Buttons to create a new project and browse templates are present.](/docs/images/deployment-guides/vercel-no-new-projects.png)
+![A Vercel dashboard with a "No projects, yet!" message. Buttons to create a new project and browse templates are present.](/docs/images/deployments/vercel-no-new-projects.png)
```

Comment:
I'm skeptical of having screenshots of another product's UI in our docs, as we'd have no idea if these got out of date, but we can leave them for now

Reply:
yeah we can definitely revisit these guides!